### PR TITLE
[luci/pass] Extend ConvertNCHWToNHWC with Elu

### PR DESCRIPTION
**Pull request by m-bronnikov**
This commit extends ConvertNCHWToNCHW pass with Elu op.

ONE-DCO-1.0-Signed-off-by: Maksim Bronnikov <max120199@gmail.com>

### CODE REVIEW

**Comment by m-bronnikov** [comment]
Motivation: I found `Elu` in some onnx model and this PR helps to eliminate large amount of `Transpose` layers.

**Comment by seanshpark** [comment]
Please add test codes in `compiler/luci/pass/src/ConvertNCHWToNHWCPass.test.cpp`

**Comment by m-bronnikov** [comment]
> Please add test codes in `compiler/luci/pass/src/ConvertNCHWToNHWCPass.test.cpp`

Done!

File: compiler/luci/pass/src/ConvertNCHWToNHWCPass.test.cpp [hunk 1]
```diff
@@ -941,6 +957,26 @@ TEST(ConvertNCHWToNHWC, Concatenation)
...
+
+  run_phase(&g.g, true, true);
+
+  check_pre_trans(g.elu->features());
+
+  auto elu_succs = loco::succs(g.elu);
+  EXPECT_EQ(1, elu_succs.size());
+  check_post_trans(*elu_succs.begin());
+
+  // Check leakyrelu shape
```
Comment (jinevening) [inline compiler/luci/pass/src/ConvertNCHWToNHWCPass.test.cpp:973]: Typo

```suggestion
  // Check elu shape
```

**Review by jinevening — APPROVED** [review]
LGTM

**Review by seanshpark — APPROVED** [review]
LGTM thank you!

**Pull request merged**